### PR TITLE
chore: show unpkg usage

### PR DESCRIPTION
**Pull request by khornberg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,17 +12,23 @@ Please see our [contributing guidelines](.github/CONTRIBUTING.md)
...
-    @britecore:registry=https://npm.pkg.github.com/
+    ```
+    npm install @britecore/ui-plugins-client
     ```
 
-2. Install by running:
+* Use in the browser by adding:
 
+    ```html
+    <script src="https://unpkg.com/@britecore/ui-plugins-client@0.9.2/dist/britecore-ui-plugins.js"></script>
```
Comment (z3by) [inline README.md:24]: why the full path, not https://unpkg.com/@britecore/ui-plugins-client.

File: README.md [hunk 2]
```diff
@@ -12,17 +12,23 @@ Please see our [contributing guidelines](.github/CONTRIBUTING.md)
 
 ### [Installing](https://docs.github.com/en/free-pro-team@latest/packages/using-github-packages-with-your-projects-ecosystem/configuring-npm-for-use-with-github-packages#installing-a-package)
```
Comment (z3by) [inline README.md:13]: no need for the link if we don't wanna use Github Package Registry.

Comment (khornberg) [reply README.md:13]: we'll get there

File: README.md (hunk 1 above)
Comment (khornberg) [reply README.md:24]: just to be explicit
we can certainly leave it off if you think that is better

**Pull request merged**